google/fingerprinters/web/scripts/updater/commuity/zabbix/versions.txt

```diff
@@ -0,0 +1,75 @@
+centos-4.0.0
```

Comment:
I couldn't run this version of the zabbix, I'm getting an error as the followings:

```
zabbix-server_1            |    161:20230310:014006.854 cannot stat SNMP trapper file "/var/lib/zabbix/snmptraps/snmptraps.log": [2] No such file or directory
```

I can run centos-5.0.0 properly, and I haven't checked the rest of the versions.

Reply:
What difference does this make

Reply:
I can run 4.0.0 after your most recent update, probably has something to do with the port #.